common/lib/dependabot/workspace/git.rb

```diff
@@ -124,14 +137,21 @@ def commit(memo = nil)
         diff = run_shell_command("git diff --cached .")
 
         msg = memo || "workspace change"
-        run_shell_command(%(git commit -m "#{msg}"), allow_unsafe_shell_command: true)
+        run_shell_command(
+          %(git commit -m "#{msg}"),
+          fingerprint: "git commit -m \"<msg>\"",
+          allow_unsafe_shell_command: true
+        )
 
         [head_sha, diff]
       end
 
       sig { params(sha: String).returns(String) }
       def reset(sha)
-        run_shell_command("git reset --hard #{sha}")
+        run_shell_command(
+          "git reset --hard #{sha}",
+          fingerprint: "git reset --hard <sha>"
+        )
```

Comment:
I can't comment below here, but the Sorbet fix is to change `kwargs` on line 164 from `T::Boolean` to `T.any(T::Boolean, String)`